src/lib/statefulForm.jsx

```diff
@@ -128,9 +135,30 @@ export default function statefulForm(mapPropsToFormConfig) {
             value: target.value
           }
         }
+        if (target.type && target.type === 'email') {
+          if (target.value && !isValidEmail(target.value)) {
+            errors.push(
+              t('account.message.error.validation', {
+                name: t(`account.form.label.${field}`)
+              })
+            )
+          }
+        }
+        if (pattern) {
```

Comment:
Since html5 allow you to use `pattern` as attributes on input, you cloud simply override the error with Constraint Validation API > https://developer.mozilla.org/fr/docs/Web/Guide/HTML/HTML5/Constraint_validation

Then for Malakof, we don't need `pattern` attribute, we could use `min` & `max`